Assets/Scripts/GameManagement/GameManager.cs

```diff
@@ -6,11 +6,11 @@
 
 public class GameManager : MonoBehaviour
 {
-    public static bool launchToMainMenu = false; //CHANGE THIS DEPENDING ON WHETHER YOU WANT
+    public static bool launchToMainMenu = true; //CHANGE THIS DEPENDING ON WHETHER YOU WANT
```

Comment:
change back to false on main, or perhaps better, conditionally compile false vs true based on if you're in the editor or not (idk whether this would cause issues for testing certain things)

Reply:
I did consider conditional compilation for this; however even in the editor I still often want to not start in the main menu so for now I think I'll just leave it as is (after changing back to false)